pkg/gofr/datasource/opentsdb/opentsdb_test.go

```diff
@@ -227,7 +229,7 @@ func TestPutErrorResponse(t *testing.T) {
 
 	err := client.PutDataPoints(context.Background(), dataPoints, "", resp)
 	require.Error(t, err)
-	require.Equal(t, "client error: 400", err.Error())
+	require.Equal(t, "openTSDB client error, status code: 400", err.Error())
```

Comment:
If you use require.Error followed by a require.Equal(t, "whatever", err.Error())

Then simply use

require.EqualError(t, err, "whatever")

But here, I would have used ErrorIs(to, err, errSentinel) + ErrorContains(t, err, "status code: 400")

Reply:
Since most errors are wrapped, I feel `require.EqualError(t, err, "whatever")` would suit the usecase better. So I will be using that.